kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue

```diff
@@ -99,19 +99,30 @@
         :selectedExercises="selectedExercises"
       />
 
-      <BottomAppBar style="z-index: 1062;">
-        <KButtonGroup>
-          <KRouterLink
-            appearance="flat-button"
-            :text="$tr('previousStep')"
-            :to="toolbarRoute"
-          />
-          <KButton
-            :text="coreString('finishAction')"
-            :disabled="loadingNewQuestions"
-            primary
-            @click="submit"
-          />
+      <BottomAppBar
+        style="z-index: 1062;"
+      >
+        <KButtonGroup
+          class="align-btn"
+        >
+          <div class="align-btn">
```

Comment:
I think that we should consider looking at how this fix might be better applied in the `KButtonGroup` KDS component itself. 

I think that [KButtonGroup](https://design-system.learningequality.org/kbuttongroup/) would fairly bring along the expectation that the buttons are vertically align. Ideally we should just be able to pass `KButtonGroup` a number of `KButton` components and see them all aligned properly.